app/src/main/java/com/iic/shopingo/api/models/ApiSimpleResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.iic.shopingo.api.models;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by assafgelber on 3/10/15.
+ */
+public class ApiSimpleResponse {
```

Comment:
I named it that way so it would be clear that it's not just a base class, but can also be instantiated on its own